### PR TITLE
Add action scopes description

### LINKED ISSUE #1523: [feature] Add layer scoped actions
Original commit: https://github.com/qgis/QGIS/commit/5b4a88f8e07e8ee1f9bfc0570f76f68717688cf2 by m-kuhn

Within the attribute table, there is a new button to trigger actions
which are not based on individual features but instead on the whole layer.
Normally they will perform actions based on all features or the selection.

In addition to this, a lot of cleanup has been done.

### PR DESCRIPTION
**Pull request by ghtmtt**
fix #1523 

not a duplicate of #2849 but a description of action scopes

### CODE REVIEW

**Comment by DelazJ** [comment]
@ghtmtt can you add more verbose title to the PRs please?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -2788,8 +2788,21 @@ where ``QGIS`` is the search term. Armed with this information, we can proceed:
...
+   1. :guilabel:`Feature Scope`: action is available when right click in the cell
+      within the attribute table
+   2. :guilabel:`Field Scope`: action is available when right click in the cell
+      within the attribute table, in the feature form and in the default action
+      button of the main toolbar
+   3. :guilabel:`Layer Scope`: action is available in the action button in the
+      attribute table toobar. Be aware that this type of action involves the entire
+      layer and not the single features
+   4. :guilabel:`Canvas`: action is available in the main action button in the
+      toolbar
```
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: All this? To be sure we are talking about the same thing, I was mentioning the order in which the items are listed in the dialog not requesting a priority order between the scopes themselves _(even though, now 😄 )_.

[the remark this answers is not in the document]
Comment (m-kuhn) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Yes, they are just requested from the registry and then listed, there is nowhere a list which could be put into an order.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -2788,8 +2788,21 @@ where ``QGIS`` is the search term. Armed with this information, we can proceed:
    layer in the legend, or right-click and choose :menuselection:`Properties`
    from the pop-up menu.
 #. Click on the :guilabel:`Actions` tab.
-#. click |signPlus| :sup:`Add a new action`.
+#. Click |signPlus| :sup:`Add a new action`.
 #. Enter a name for the action, for example ``Google Search``.
```
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: what do you mean exactly? I just found this small typo and changed the capital letter ;)

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: And drew my attention to it ;)
I meant that in the dialog there are also options like `Type`, `Capture output`,`short name` or `icon` that we should maybe tell the user how to fill them if we go the step by step  example way...

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: It Works. I guess it might be using the application set in the path

**Comment by ghtmtt** [comment]
@DelazJ can we merge this?

**Comment by DelazJ** [comment]
I think the global update can be done in a next step.

**Pull request merged**